src/shared/constants/realms.ts

```diff
@@ -54,14 +61,20 @@ export const FIGMA_FACTOR = {
   Y: ISLAND_BOX.height / FIGMA_WORKSPACE_SIZE.height,
 }
 
-export const REALMS_MAP_DATA: RealmMapData[] = [
+export const BASE_REALMS_MAP_DATA: RealmMapData[] = [
   realm4,
   realm7,
   realm9,
   realm19,
   realm22,
 ]
 
+export const ADDITIONAL_REALMS_MAP_DATA: RealmMapData[] = [realm16]
+
+export const REALMS_MAP_DATA = process.env.SHOW_ADDITIONAL_REALMS
```

Comment:
it must be `process.env.SHOW_ADDITIONAL_REALMS === "true"` becuase env vars are strings so `SHOW_ADDITIONAL_REALMS="false"`will break it